app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
+    @comment.save
```

Comment:
That argument that you're passing to render is looking for a specific view template, so the path needs to match the path in your views folder (i.e. switch `:'job/show'` to `:'jobs/show'`).

One additional note: would probably switch the syntax on lines 4-5 to the following:

```ruby
@job     = Job.find(params[:job_id])
@comment = @job.comments.new(comment_params)
```

Has the added advantage of defining `@job` for use in the `jobs/show` view that you're hoping to render on fail.